website/docs/reference/sinks/influxdb_metrics.md.erb

```diff
@@ -0,0 +1,80 @@
+## Env Vars
+
+<%= fields(component.env_vars_list, heading_depth: 3) %>
+
+<%- end -%>
+<%- if component.output -%>
+## Output
+
+<%= component_output(component, component.output) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
+### Configuration example
+
+#### InfluxDB v1
+```toml
+[sinks.my_sink_id]
+  type = "influxdb_metrics"
+  namespace = "service"
+  endpoint = "https://us-west-2-1.aws.cloud1.influxdata.com"
+  database = "vector-database"
+  consistency = "one"
+  retention_policy_name = "one_day_only"
+  username = "vector-source"
+  password = "${INFLUXDB_PASSWORD_ENV_VAR}"
+```
+
+#### InfluxDB v2
+```toml
+[sinks.my_sink_id]
+  type = "influxdb_metrics"
+  namespace = "service"
+  endpoint = "https://us-west-2-1.aws.cloud2.influxdata.com"
+  org = "my-org"
+  bucket = "my-bucket"
+  token = "${INFLUXDB_TOKEN_ENV_VAR}"
+```
+
+### Metric Types
+
+InfluxDB uses [line protocol][urls.influxdb_line_protocol] to write data points. It is a text-based format that provides the measurement, tag set, field set, and timestamp of a data point.
+
+The following matrix outlines how Vector metric types are mapped into InfluxDB Line Protocol fields.
+
+| Vector Metrics | Line Protocol Fields                             | Example                                                                                                                                                 |
+|----------------|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Counter        | value                                            | ns.total,metric_type=counter value=1.5 1542182950000000011                                                                                              |
+| Gauge          | value                                            | ns.meter,metric_type=gauge,normal_tag=value,true_tag=true value=-1.5 1542182950000000011                                                                |
+| Set            | value                                            | ns.users,metric_type=set,normal_tag=value,true_tag=true value=2 154218295000000001                                                                      |
+| Histogram      | buckets, count, sum                              | ns.requests,metric_type=histogram,normal_tag=value,true_tag=true bucket_1=1i,bucket_2.1=2i,bucket_3=3i,count=6i,sum=12.5 1542182950000000011            |
+| Summary        | quantiles, count, sum                            | ns.requests_sum,metric_type=summary,normal_tag=value,true_tag=true count=6i,quantile_0.01=1.5,quantile_0.5=2,quantile_0.99=3,sum=12 1542182950000000011 |
+| Distribution   | min, max, median, avg, sum, count, quantile 0.95 | ns.sparse_stats,metric_type=distribution avg=3,count=10,max=4,median=3,min=1,quantile_0.95=4,sum=30 1542182950000000011                                 |
+
```

Comment:
Would it be slightly easier to read if the examples were marked as code? Like this:

| Vector Metrics | Line Protocol Fields                             | Example                                                                                                                                                 |
|----------------|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
| Counter        | `value`                                            | `ns.total,metric_type=counter value=1.5 1542182950000000011`                                                                                              |
| Gauge          | value                                            | `ns.meter,metric_type=gauge,normal_tag=value,true_tag=true value=-1.5 1542182950000000011`                                                                |
| ... |

?

Reply:
Yeah, it's better. Changed.